posts/seurat-cosmx-basics/index.qmd

```diff
@@ -59,11 +58,90 @@ Adjust globals option to avoid an error exceeding max allowed size. We've found
+There are two approaches to export CosMx SMI data from AtoMx SIP and load them into a Seurat object. Data can be exported from AtoMx SIP directly into a Seurat object. Alternatively, raw data can be exported as flat files and then loaded into a Seurat object. Both approaches are demonstrated below.
+
+::: {.panel-tabset}
+
+## Export Seurat object
+
+To export CosMx data from AtoMx SIP directly into a Seurat object, select "Export a Seurat Object" from the export module as shown in @fig-seurat-export-option. Be sure to export the Seurat object with transcript coordinates and polygon coordinates included to access all of the functionality below.
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-seurat-export-option"
+#| fig-cap: "Screenshot showing selection of Seurat object for export from AtoMx SIP."
+
+knitr::include_graphics("./figures/AtoMx_seuratExport_image.png")
+```
+
+Once export is complete, download the file
+using your SFTP application (_e.g._, cyberduck, FileZilla, WinSCP).
+
+You can then load the Seurat object into R as follows. The Seurat object used for the demonstration here is available on Box.com [here](https://nanostring.box.com/s/y7ielc54qw1g1f5x98hxczdo8tezobbg){target="_blank"}.
 ```{r load_seuratObject, message=FALSE, eval = FALSE}
-seu.obj <- readRDS("seurat_object.Rds")
+seu.obj <- readRDS("/path/to/file/seurat_object.Rds")
+```
+
+
+## Export flat files
+
+To export CosMx data from AtoMx SIP into flat files, select all five flat files from the export module as shown in @fig-flat-file-export-option. For more information about the flat file formats, please see [this post](../flat-file-exports/flat-files-compare.qmd). When taking this approach, some tertiary analyses run in AtoMx SIP may not be included. For example, any PCA or UMAP dimension reductions performed in AtoMx SIP do not get exported in the flat files and so will not be present in the Seurat object when generated from flat files.
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-flat-file-export-option"
+#| fig-cap: "Screenshot showing selection of flat files for export from AtoMx SIP."
+
+knitr::include_graphics("./figures/AtoMx_flatFileExport_image.png")
+```
+
+Once export is complete, download the files
+using your SFTP application (_e.g._, cyberduck, FileZilla, WinSCP).
+
+Load the flat files into a Seurat object using the `LoadNanostring` function from the Seurat package. Within the Seurat object, each slide is stored as a separate 'image' or 'fov'. This is an unfortunate naming convention difference between CosMx nomenclature and the Seurat package. What Seurat refers to as an ‘fov’ is what NanoString refers to as a slide. When you load flat files into a Seurat object, you need to name the Seurat fov / CosMx slide with the argument `fov = `. This name must be a character, not a numeric. Here we use "slide1", but more descriptive names are also possible.
```

Comment:
...between CosMx SMI nomenclature...

Reply:
How does one refer to just a single SMI FOV then?

Reply:
"SMI" added and a note about how to plot single SMI FOVs added.